apps/src/p5lab/P5Lab.js

```diff
@@ -794,7 +794,6 @@ export default class P5Lab {
     clearMarks(DRAW_LOOP_START);
     clearMeasures(DRAW_LOOP_MEASURE);
     this.spriteTotalCount = 0;
-
```

Comment:
nit: mind reverting this just so the file isn't included in the PR?

Reply:
Sure! Will do.